### PR TITLE
I changed something in your code

### PR DESCRIPTION
**Pull request by MagbuleB**

### CODE REVIEW

**Comment by MagbuleB** [comment]
Anda's change

**Comment by MagbuleB** [comment]
Anda change something

File: day39_Recap/Triangle.java [hunk 1]
```diff
@@ -8,4 +8,6 @@ public class Triangle {
     //Im doing some changes
 
     // great! we dine it:)
+
+    //Im changing something in branch
```
Comment (sofiiberlin) [inline day39_Recap/Triangle.java:12]: great thank you!

**Review by sofiiberlin — APPROVED** [review]

**Review by sofiiberlin — APPROVED** [review]
well done

**Review by AndaKrasniqi — APPROVED** [review]
its okay

**Review by AndaKrasniqi — APPROVED** [review]
its okay

**Pull request closed without merging**